packages/react-input/src/stories/InputAppearance.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { Label } from '@fluentui/react-label';
+import { useId } from '@fluentui/react-utilities';
+import { makeStyles } from '@fluentui/react-make-styles';
+import { Input } from '../index';
+
+const useStyles = makeStyles({
+  root: {
+    '& label': { display: 'block', paddingBottom: '2px' },
+    '& label:not(:first-child)': { paddingTop: '20px' },
+  },
+});
+
+export const Appearance = () => {
+  const outlineId = useId('input-outline');
+  const underlineId = useId('input-underline');
+  const filledLighterId = useId('input-filledLighter');
+  const filledDarkerId = useId('input-filledDarker');
```

Comment:
To keep the examples simpler, could we just use hardcoded IDs? I don't think useId is really necessary here?
```
      <Label htmlFor="input-outline">Outline (default)</Label>
      <Input appearance="outline" id="input-outline" />
```

Reply:
No, because if the examples are copy-pasted this tends to result in duplicate IDs.

Reply:
No, because if the stories are copy-pasted (either as a starting point for an additional story, or possibly into someone's app), using hardcoded IDs tends to result in duplicates. Using `useId` prevents the duplicate issue and demonstrates best practices for actual apps.